Project/Source/Modules/ModuleScene.cpp

```diff
@@ -94,20 +97,14 @@ bool ModuleScene::Start() {
 	App->files->CreateFolder(NAVMESH_PATH);
 #endif
 
+	App->scene->CreateEmptyScene();
```

Comment:
I'm assuming this is necessary for the Resources Rework? Otherwise, it's an unnecessary call outside the else statement you've deleted this from.

Reply:
Yeah, it is important that the scene is never null.